src/components/velocity.js

```diff
@@ -0,0 +1,25 @@
+var registerComponent = require('../core/component').registerComponent;
+
+/**
+ * Velocity component.
+ *
+ * Updates object's position at each tick.
+ */
+module.exports = registerComponent('velocity', {
+  dependencies: ['position'],
```

Comment:
I don't think we need to specify this since it is a default component on all entities